### DISCUSSION TITLE
Airflow UI log is different from real log file

### DISCUSSION THREAD

**Discussion by yjs03057** [apache/airflow#27925] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

As the issue title, Airflow ui shows different timezone with log file.
And it is also different with mini-map.

"mini-map with KST timezone" - this is right log
![image](https://user-images.githubusercontent.com/43867737/202333994-df039e67-83e0-46f1-bd8f-7d779679ae2e.png)
"ui log with KST timezone" -this is WRONG
![image](https://user-images.githubusercontent.com/43867737/202334101-2204e173-867f-4601-b972-b65af8f12dc1.png)

why it convert once more in the ui log page though it has same timezone? I just wanna see like mini-map.....

++ My airflow version is 2.3.4

### Operating System

Ubuntu

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by notatallshaw-gts** [comment]
Have you checked the release notes for 2.3.4: https://github.com/apache/airflow/blob/main/RELEASE_NOTES.rst#added-new-config-logginglog_formatter_class-to-fix-timezone-display-for-logs-on-ui-24811 ?

**Comment by yjs03057** [comment]
I'm not using custom formatter class... And I didn't change anything about log formatter config from default value. Do I need to make custom formatter to fix this?

++) My current log formatter setting
![image](https://user-images.githubusercontent.com/43867737/202618006-7dfa070b-a67a-4ef1-977b-339be5dce4ef.png)

**Comment by notatallshaw-gts** [comment]
Not sure, was just a suggestion. I am not having this issue on 2.3.4. I did however have it, or something similar on 2.3.3, at least for New York time zone, when I moved to 2.3.4 (without changing anything) everything worked.

**Comment by potiuk** [comment]
Not sure if this is a real issue. Converting to a dicscussion until @yjs03057 confirms after migration
